pkg/settings/setting.go

```diff
@@ -93,3 +94,13 @@ func getSystemImages() string {
 	}
 	return string(data)
 }
+
+func getSystemNamespaces() string {
+	systemNamespaces := []string{"kube-system", "kube-public", "cattle-system", "cattle-alerting", "cattle-logging",
```

Comment:
Can this just be a comma separated list, not json encoded

Reply:
@ibuildthecloud sure